test/e2e/certificates_test.go

```diff
@@ -267,6 +270,71 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			err = verifyCertificate(ctx, certName, ns.Name, certDomain)
 			Expect(err).NotTo(HaveOccurred())
 		})
+
+		It("should obtain a valid LetsEncrypt certificate in GCP mint/passthrough clusters-65035", func() {
+
+			By("Getting Infrastructure object")
+			infra, err := configClient.Infrastructures().Get(ctx, "cluster", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			if infra.Status.PlatformStatus.Type != configv1.GCPPlatformType {
+				Skip("Skip this case for current plaform, it's not gcp.")
+			}
+
+			By("Check cloud credential in cluster")
+			_, err = loader.KubeClient.CoreV1().Secrets("kube-system").Get(ctx, "gcp-credentials", metav1.GetOptions{})
+			if err != nil {
+				Skip("Skipping for the cluster without credential in cluster")
+			}
+
+			By("Creating CredentialsRequest object")
+			loader.CreateFromFile(testassets.ReadFile, filepath.Join("testdata", "credentials", "credentialsrequest_gcp.yaml"), "")
+
+			By("Waiting for cloud secret to be available")
+			err = wait.PollImmediate(PollInterval, TestTimeout, func() (bool, error) {
+				_, err := loader.KubeClient.CoreV1().Secrets("cert-manager").Get(ctx, "gcp-creds", metav1.GetOptions{})
+				if err != nil {
+					return true, nil
+				}
+				return false, nil
+			})
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Configure cert-manager to use credential, setting this credential secret name in subscription object")
+			// The name is defined cloud credential in cluster ns: kube-system
```

Comment:
Incorrect. This one is not defined in kube-system. Please see explanations again which were given in https://github.com/openshift/cert-manager-operator/pull/166#discussion_r1436889237 and sample comment was given in https://github.com/openshift/cert-manager-operator/pull/166#discussion_r1437348369